lib/integrations/openai/processor_service.rb

```diff
@@ -10,63 +10,44 @@ def summarize_message
   end
 
   def rephrase_message
-    make_api_call(rephrase_body)
+    make_api_call(build_api_call_body("#{AGENT_INSTRUCTION} Please rephrase the following response. " \
+                                      "#{LANGUAGE_INSTRUCTION}"))
   end
 
   def fix_spelling_grammar_message
-    make_api_call(fix_spelling_grammar_body)
+    make_api_call(build_api_call_body("#{AGENT_INSTRUCTION} Please fix the spelling and grammar of the following response. " \
+                                      "#{LANGUAGE_INSTRUCTION}"))
   end
 
   def shorten_message
-    make_api_call(shorten_body)
+    make_api_call(build_api_call_body("#{AGENT_INSTRUCTION} Please shorten the following response. " \
+                                      "#{LANGUAGE_INSTRUCTION}"))
   end
 
   def expand_message
-    make_api_call(expand_body)
+    make_api_call(build_api_call_body("#{AGENT_INSTRUCTION} Please expand the following response. " \
+                                      "#{LANGUAGE_INSTRUCTION}"))
   end
 
   def make_friendly_message
-    make_api_call(make_friendly_body)
+    make_api_call(build_api_call_body("#{AGENT_INSTRUCTION} Please make the following response more friendly. " \
+                                      "#{LANGUAGE_INSTRUCTION}"))
   end
 
   def make_formal_message
-    make_api_call(make_formal_body)
+    make_api_call(build_api_call_body("#{AGENT_INSTRUCTION} Please make the following response more formal. " \
+                                      "#{LANGUAGE_INSTRUCTION}"))
   end
 
   def simplify_message
-    make_api_call(simplify_body)
+    make_api_call(build_api_call_body("#{AGENT_INSTRUCTION} Please simplify the following response. " \
+                                      "#{LANGUAGE_INSTRUCTION}"))
   end
 
   private
 
-  def rephrase_body
-    build_api_call_body("#{AGENT_INSTRUCTION} Please rephrase the following response. " \
-                        "#{LANGUAGE_INSTRUCTION}")
-  end
-
-  def fix_spelling_grammar_body
-    build_api_call_body("#{AGENT_INSTRUCTION} Please fix the spelling and grammar of the following response. " \
-                        "#{LANGUAGE_INSTRUCTION}")
-  end
-
-  def shorten_body
-    build_api_call_body("#{AGENT_INSTRUCTION} Please shorten the following response. #{LANGUAGE_INSTRUCTION}")
-  end
-
-  def expand_body
-    build_api_call_body("#{AGENT_INSTRUCTION} Please expand the following response. #{LANGUAGE_INSTRUCTION}")
-  end
-
-  def make_friendly_body
-    build_api_call_body("#{AGENT_INSTRUCTION} Please make the following response more friendly. #{LANGUAGE_INSTRUCTION}")
-  end
-
-  def make_formal_body
-    build_api_call_body("#{AGENT_INSTRUCTION} Please make the following response more formal. #{LANGUAGE_INSTRUCTION}")
-  end
-
-  def simplify_body
-    build_api_call_body("#{AGENT_INSTRUCTION} Please simplify the following response. #{LANGUAGE_INSTRUCTION}")
+  def prompts_from_file(file_name)
```

Comment:
```suggestion
  def prompts_from_file(file_name)
```

This name is too close to the one used in enterprise, can be confusing, maybe we can let this be `prompt_from_file` and for the other one, we can override that

```rb
def prompt_from_file(file_name, enterprise = False)
  path = enterprise ? 'enterprise/lib/enterprise/integrations/openai_prompts' : 'lib/integrations/openai_prompts'
  Rails.root.join(path, "#{file_name}.txt").read
end
```

CC: @sojan-official

Reply:
Sounds good. 
- Also, is there a case where we have separate prompts for open source/enterprise?
- Do we need to keep prompts under enterprise? 

if not we can have a unified folder for all the prompts.

Reply:
> Also, is there a case where we have separate prompts for open source/enterprise?

Yes. We use different prompts for the summary in EE and OS.